bootstrap

```diff
@@ -4,6 +4,14 @@
 # Any flags will be passed on to CMake, e.g.,
 # ./bootstrap -DCMAKE_BUILD_TYPE="Debug"
 
+## Check to make cmake is installed, if not offer to install if apt is
+## available or exit
+if ! command -v cmake >/dev/null 2>&1; then
+	echo "cmake is required to bootstrap"
+	command -v apt >/dev/null 2>&1 && (sudo apt install cmake || exit)
```

Comment:
We don't want to check for apt or try to install it here, that's too Debian/Ubuntu-centric since we support RedHat-based distros, Windows, OSX, etc and we don't want to include checks for all of those.

I'd recommend just printing out something like:
```
cmake is required to build TimescaleDB. Please install via your system's preferred method.
```

Then exiting with a non-0 exit code

Reply:
Cool and done.